From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: P4ulBr4ndt <79166045+P4ulBr4ndt@users.noreply.github.com>
Date: Mon, 28 Aug 2023 13:05:38 +0200
Subject: [PATCH] 5712: Fixes CAN_MO addresses for Tricore TC176X mcus

Fix tricore TC176X CAN_MO addresses

Fix tricore TC176X CAN_MO addresses
---
 .../tricore/data/languages/tc176x.pspec       | 1024 ++++++++---------
 1 file changed, 512 insertions(+), 512 deletions(-)

diff --git a/Ghidra/Processors/tricore/data/languages/tc176x.pspec b/Ghidra/Processors/tricore/data/languages/tc176x.pspec
index 891426f83..e5d96cdc2 100644
--- a/Ghidra/Processors/tricore/data/languages/tc176x.pspec
+++ b/Ghidra/Processors/tricore/data/languages/tc176x.pspec
@@ -669,518 +669,518 @@
     <symbol name="CAN_NBTR1" address="0xf0004310" entry="false"/>
     <symbol name="CAN_NECNT1" address="0xf0004314" entry="false"/>
     <symbol name="CAN_NFCR1" address="0xf0004318" entry="false"/>
-    <symbol name="CAN_MOFCR0" address="0xf0004600" entry="false"/>
-    <symbol name="CAN_MOFCR1" address="0xf0004620" entry="false"/>
-    <symbol name="CAN_MOFCR2" address="0xf0004640" entry="false"/>
-    <symbol name="CAN_MOFCR3" address="0xf0004660" entry="false"/>
-    <symbol name="CAN_MOFCR4" address="0xf0004680" entry="false"/>
-    <symbol name="CAN_MOFCR5" address="0xf00046a0" entry="false"/>
-    <symbol name="CAN_MOFCR6" address="0xf00046c0" entry="false"/>
-    <symbol name="CAN_MOFCR7" address="0xf00046e0" entry="false"/>
-    <symbol name="CAN_MOFCR8" address="0xf0004700" entry="false"/>
-    <symbol name="CAN_MOFCR9" address="0xf0004720" entry="false"/>
-    <symbol name="CAN_MOFCR10" address="0xf0004740" entry="false"/>
-    <symbol name="CAN_MOFCR11" address="0xf0004760" entry="false"/>
-    <symbol name="CAN_MOFCR12" address="0xf0004780" entry="false"/>
-    <symbol name="CAN_MOFCR13" address="0xf00047a0" entry="false"/>
-    <symbol name="CAN_MOFCR14" address="0xf00047c0" entry="false"/>
-    <symbol name="CAN_MOFCR15" address="0xf00047e0" entry="false"/>
-    <symbol name="CAN_MOFCR16" address="0xf0004800" entry="false"/>
-    <symbol name="CAN_MOFCR17" address="0xf0004820" entry="false"/>
-    <symbol name="CAN_MOFCR18" address="0xf0004840" entry="false"/>
-    <symbol name="CAN_MOFCR19" address="0xf0004860" entry="false"/>
-    <symbol name="CAN_MOFCR20" address="0xf0004880" entry="false"/>
-    <symbol name="CAN_MOFCR21" address="0xf00048a0" entry="false"/>
-    <symbol name="CAN_MOFCR22" address="0xf00048c0" entry="false"/>
-    <symbol name="CAN_MOFCR23" address="0xf00048e0" entry="false"/>
-    <symbol name="CAN_MOFCR24" address="0xf0004900" entry="false"/>
-    <symbol name="CAN_MOFCR25" address="0xf0004920" entry="false"/>
-    <symbol name="CAN_MOFCR26" address="0xf0004940" entry="false"/>
-    <symbol name="CAN_MOFCR27" address="0xf0004960" entry="false"/>
-    <symbol name="CAN_MOFCR28" address="0xf0004980" entry="false"/>
-    <symbol name="CAN_MOFCR29" address="0xf00049a0" entry="false"/>
-    <symbol name="CAN_MOFCR30" address="0xf00049c0" entry="false"/>
-    <symbol name="CAN_MOFCR31" address="0xf00049e0" entry="false"/>
-    <symbol name="CAN_MOFCR32" address="0xf0004a00" entry="false"/>
-    <symbol name="CAN_MOFCR33" address="0xf0004a20" entry="false"/>
-    <symbol name="CAN_MOFCR34" address="0xf0004a40" entry="false"/>
-    <symbol name="CAN_MOFCR35" address="0xf0004a60" entry="false"/>
-    <symbol name="CAN_MOFCR36" address="0xf0004a80" entry="false"/>
-    <symbol name="CAN_MOFCR37" address="0xf0004aa0" entry="false"/>
-    <symbol name="CAN_MOFCR38" address="0xf0004ac0" entry="false"/>
-    <symbol name="CAN_MOFCR39" address="0xf0004ae0" entry="false"/>
-    <symbol name="CAN_MOFCR40" address="0xf0004b00" entry="false"/>
-    <symbol name="CAN_MOFCR41" address="0xf0004b20" entry="false"/>
-    <symbol name="CAN_MOFCR42" address="0xf0004b40" entry="false"/>
-    <symbol name="CAN_MOFCR43" address="0xf0004b60" entry="false"/>
-    <symbol name="CAN_MOFCR44" address="0xf0004b80" entry="false"/>
-    <symbol name="CAN_MOFCR45" address="0xf0004ba0" entry="false"/>
-    <symbol name="CAN_MOFCR46" address="0xf0004bc0" entry="false"/>
-    <symbol name="CAN_MOFCR47" address="0xf0004be0" entry="false"/>
-    <symbol name="CAN_MOFCR48" address="0xf0004c00" entry="false"/>
-    <symbol name="CAN_MOFCR49" address="0xf0004c20" entry="false"/>
-    <symbol name="CAN_MOFCR50" address="0xf0004c40" entry="false"/>
-    <symbol name="CAN_MOFCR51" address="0xf0004c60" entry="false"/>
-    <symbol name="CAN_MOFCR52" address="0xf0004c80" entry="false"/>
-    <symbol name="CAN_MOFCR53" address="0xf0004ca0" entry="false"/>
-    <symbol name="CAN_MOFCR54" address="0xf0004cc0" entry="false"/>
-    <symbol name="CAN_MOFCR55" address="0xf0004ce0" entry="false"/>
-    <symbol name="CAN_MOFCR56" address="0xf0004d00" entry="false"/>
-    <symbol name="CAN_MOFCR57" address="0xf0004d20" entry="false"/>
-    <symbol name="CAN_MOFCR58" address="0xf0004d40" entry="false"/>
-    <symbol name="CAN_MOFCR59" address="0xf0004d60" entry="false"/>
-    <symbol name="CAN_MOFCR60" address="0xf0004d80" entry="false"/>
-    <symbol name="CAN_MOFCR61" address="0xf0004da0" entry="false"/>
-    <symbol name="CAN_MOFCR62" address="0xf0004dc0" entry="false"/>
-    <symbol name="CAN_MOFCR63" address="0xf0004de0" entry="false"/>
-    <symbol name="CAN_MOFGPR0" address="0xf0004604" entry="false"/>
-    <symbol name="CAN_MOFGPR1" address="0xf0004624" entry="false"/>
-    <symbol name="CAN_MOFGPR2" address="0xf0004644" entry="false"/>
-    <symbol name="CAN_MOFGPR3" address="0xf0004664" entry="false"/>
-    <symbol name="CAN_MOFGPR4" address="0xf0004684" entry="false"/>
-    <symbol name="CAN_MOFGPR5" address="0xf00046a4" entry="false"/>
-    <symbol name="CAN_MOFGPR6" address="0xf00046c4" entry="false"/>
-    <symbol name="CAN_MOFGPR7" address="0xf00046e4" entry="false"/>
-    <symbol name="CAN_MOFGPR8" address="0xf0004704" entry="false"/>
-    <symbol name="CAN_MOFGPR9" address="0xf0004724" entry="false"/>
-    <symbol name="CAN_MOFGPR10" address="0xf0004744" entry="false"/>
-    <symbol name="CAN_MOFGPR11" address="0xf0004764" entry="false"/>
-    <symbol name="CAN_MOFGPR12" address="0xf0004784" entry="false"/>
-    <symbol name="CAN_MOFGPR13" address="0xf00047a4" entry="false"/>
-    <symbol name="CAN_MOFGPR14" address="0xf00047c4" entry="false"/>
-    <symbol name="CAN_MOFGPR15" address="0xf00047e4" entry="false"/>
-    <symbol name="CAN_MOFGPR16" address="0xf0004804" entry="false"/>
-    <symbol name="CAN_MOFGPR17" address="0xf0004824" entry="false"/>
-    <symbol name="CAN_MOFGPR18" address="0xf0004844" entry="false"/>
-    <symbol name="CAN_MOFGPR19" address="0xf0004864" entry="false"/>
-    <symbol name="CAN_MOFGPR20" address="0xf0004884" entry="false"/>
-    <symbol name="CAN_MOFGPR21" address="0xf00048a4" entry="false"/>
-    <symbol name="CAN_MOFGPR22" address="0xf00048c4" entry="false"/>
-    <symbol name="CAN_MOFGPR23" address="0xf00048e4" entry="false"/>
-    <symbol name="CAN_MOFGPR24" address="0xf0004904" entry="false"/>
-    <symbol name="CAN_MOFGPR25" address="0xf0004924" entry="false"/>
-    <symbol name="CAN_MOFGPR26" address="0xf0004944" entry="false"/>
-    <symbol name="CAN_MOFGPR27" address="0xf0004964" entry="false"/>
-    <symbol name="CAN_MOFGPR28" address="0xf0004984" entry="false"/>
-    <symbol name="CAN_MOFGPR29" address="0xf00049a4" entry="false"/>
-    <symbol name="CAN_MOFGPR30" address="0xf00049c4" entry="false"/>
-    <symbol name="CAN_MOFGPR31" address="0xf00049e4" entry="false"/>
-    <symbol name="CAN_MOFGPR32" address="0xf0004a04" entry="false"/>
-    <symbol name="CAN_MOFGPR33" address="0xf0004a24" entry="false"/>
-    <symbol name="CAN_MOFGPR34" address="0xf0004a44" entry="false"/>
-    <symbol name="CAN_MOFGPR35" address="0xf0004a64" entry="false"/>
-    <symbol name="CAN_MOFGPR36" address="0xf0004a84" entry="false"/>
-    <symbol name="CAN_MOFGPR37" address="0xf0004aa4" entry="false"/>
-    <symbol name="CAN_MOFGPR38" address="0xf0004ac4" entry="false"/>
-    <symbol name="CAN_MOFGPR39" address="0xf0004ae4" entry="false"/>
-    <symbol name="CAN_MOFGPR40" address="0xf0004b04" entry="false"/>
-    <symbol name="CAN_MOFGPR41" address="0xf0004b24" entry="false"/>
-    <symbol name="CAN_MOFGPR42" address="0xf0004b44" entry="false"/>
-    <symbol name="CAN_MOFGPR43" address="0xf0004b64" entry="false"/>
-    <symbol name="CAN_MOFGPR44" address="0xf0004b84" entry="false"/>
-    <symbol name="CAN_MOFGPR45" address="0xf0004ba4" entry="false"/>
-    <symbol name="CAN_MOFGPR46" address="0xf0004bc4" entry="false"/>
-    <symbol name="CAN_MOFGPR47" address="0xf0004be4" entry="false"/>
-    <symbol name="CAN_MOFGPR48" address="0xf0004c04" entry="false"/>
-    <symbol name="CAN_MOFGPR49" address="0xf0004c24" entry="false"/>
-    <symbol name="CAN_MOFGPR50" address="0xf0004c44" entry="false"/>
-    <symbol name="CAN_MOFGPR51" address="0xf0004c64" entry="false"/>
-    <symbol name="CAN_MOFGPR52" address="0xf0004c84" entry="false"/>
-    <symbol name="CAN_MOFGPR53" address="0xf0004ca4" entry="false"/>
-    <symbol name="CAN_MOFGPR54" address="0xf0004cc4" entry="false"/>
-    <symbol name="CAN_MOFGPR55" address="0xf0004ce4" entry="false"/>
-    <symbol name="CAN_MOFGPR56" address="0xf0004d04" entry="false"/>
-    <symbol name="CAN_MOFGPR57" address="0xf0004d24" entry="false"/>
-    <symbol name="CAN_MOFGPR58" address="0xf0004d44" entry="false"/>
-    <symbol name="CAN_MOFGPR59" address="0xf0004d64" entry="false"/>
-    <symbol name="CAN_MOFGPR60" address="0xf0004d84" entry="false"/>
-    <symbol name="CAN_MOFGPR61" address="0xf0004da4" entry="false"/>
-    <symbol name="CAN_MOFGPR62" address="0xf0004dc4" entry="false"/>
-    <symbol name="CAN_MOFGPR63" address="0xf0004de4" entry="false"/>
-    <symbol name="CAN_MOIPR0" address="0xf0004608" entry="false"/>
-    <symbol name="CAN_MOIPR1" address="0xf0004628" entry="false"/>
-    <symbol name="CAN_MOIPR2" address="0xf0004648" entry="false"/>
-    <symbol name="CAN_MOIPR3" address="0xf0004668" entry="false"/>
-    <symbol name="CAN_MOIPR4" address="0xf0004688" entry="false"/>
-    <symbol name="CAN_MOIPR5" address="0xf00046a8" entry="false"/>
-    <symbol name="CAN_MOIPR6" address="0xf00046c8" entry="false"/>
-    <symbol name="CAN_MOIPR7" address="0xf00046e8" entry="false"/>
-    <symbol name="CAN_MOIPR8" address="0xf0004708" entry="false"/>
-    <symbol name="CAN_MOIPR9" address="0xf0004728" entry="false"/>
-    <symbol name="CAN_MOIPR10" address="0xf0004748" entry="false"/>
-    <symbol name="CAN_MOIPR11" address="0xf0004768" entry="false"/>
-    <symbol name="CAN_MOIPR12" address="0xf0004788" entry="false"/>
-    <symbol name="CAN_MOIPR13" address="0xf00047a8" entry="false"/>
-    <symbol name="CAN_MOIPR14" address="0xf00047c8" entry="false"/>
-    <symbol name="CAN_MOIPR15" address="0xf00047e8" entry="false"/>
-    <symbol name="CAN_MOIPR16" address="0xf0004808" entry="false"/>
-    <symbol name="CAN_MOIPR17" address="0xf0004828" entry="false"/>
-    <symbol name="CAN_MOIPR18" address="0xf0004848" entry="false"/>
-    <symbol name="CAN_MOIPR19" address="0xf0004868" entry="false"/>
-    <symbol name="CAN_MOIPR20" address="0xf0004888" entry="false"/>
-    <symbol name="CAN_MOIPR21" address="0xf00048a8" entry="false"/>
-    <symbol name="CAN_MOIPR22" address="0xf00048c8" entry="false"/>
-    <symbol name="CAN_MOIPR23" address="0xf00048e8" entry="false"/>
-    <symbol name="CAN_MOIPR24" address="0xf0004908" entry="false"/>
-    <symbol name="CAN_MOIPR25" address="0xf0004928" entry="false"/>
-    <symbol name="CAN_MOIPR26" address="0xf0004948" entry="false"/>
-    <symbol name="CAN_MOIPR27" address="0xf0004968" entry="false"/>
-    <symbol name="CAN_MOIPR28" address="0xf0004988" entry="false"/>
-    <symbol name="CAN_MOIPR29" address="0xf00049a8" entry="false"/>
-    <symbol name="CAN_MOIPR30" address="0xf00049c8" entry="false"/>
-    <symbol name="CAN_MOIPR31" address="0xf00049e8" entry="false"/>
-    <symbol name="CAN_MOIPR32" address="0xf0004a08" entry="false"/>
-    <symbol name="CAN_MOIPR33" address="0xf0004a28" entry="false"/>
-    <symbol name="CAN_MOIPR34" address="0xf0004a48" entry="false"/>
-    <symbol name="CAN_MOIPR35" address="0xf0004a68" entry="false"/>
-    <symbol name="CAN_MOIPR36" address="0xf0004a88" entry="false"/>
-    <symbol name="CAN_MOIPR37" address="0xf0004aa8" entry="false"/>
-    <symbol name="CAN_MOIPR38" address="0xf0004ac8" entry="false"/>
-    <symbol name="CAN_MOIPR39" address="0xf0004ae8" entry="false"/>
-    <symbol name="CAN_MOIPR40" address="0xf0004b08" entry="false"/>
-    <symbol name="CAN_MOIPR41" address="0xf0004b28" entry="false"/>
-    <symbol name="CAN_MOIPR42" address="0xf0004b48" entry="false"/>
-    <symbol name="CAN_MOIPR43" address="0xf0004b68" entry="false"/>
-    <symbol name="CAN_MOIPR44" address="0xf0004b88" entry="false"/>
-    <symbol name="CAN_MOIPR45" address="0xf0004ba8" entry="false"/>
-    <symbol name="CAN_MOIPR46" address="0xf0004bc8" entry="false"/>
-    <symbol name="CAN_MOIPR47" address="0xf0004be8" entry="false"/>
-    <symbol name="CAN_MOIPR48" address="0xf0004c08" entry="false"/>
-    <symbol name="CAN_MOIPR49" address="0xf0004c28" entry="false"/>
-    <symbol name="CAN_MOIPR50" address="0xf0004c48" entry="false"/>
-    <symbol name="CAN_MOIPR51" address="0xf0004c68" entry="false"/>
-    <symbol name="CAN_MOIPR52" address="0xf0004c88" entry="false"/>
-    <symbol name="CAN_MOIPR53" address="0xf0004ca8" entry="false"/>
-    <symbol name="CAN_MOIPR54" address="0xf0004cc8" entry="false"/>
-    <symbol name="CAN_MOIPR55" address="0xf0004ce8" entry="false"/>
-    <symbol name="CAN_MOIPR56" address="0xf0004d08" entry="false"/>
-    <symbol name="CAN_MOIPR57" address="0xf0004d28" entry="false"/>
-    <symbol name="CAN_MOIPR58" address="0xf0004d48" entry="false"/>
-    <symbol name="CAN_MOIPR59" address="0xf0004d68" entry="false"/>
-    <symbol name="CAN_MOIPR60" address="0xf0004d88" entry="false"/>
-    <symbol name="CAN_MOIPR61" address="0xf0004da8" entry="false"/>
-    <symbol name="CAN_MOIPR62" address="0xf0004dc8" entry="false"/>
-    <symbol name="CAN_MOIPR63" address="0xf0004de8" entry="false"/>
-    <symbol name="CAN_MOAMR0" address="0xf000460c" entry="false"/>
-    <symbol name="CAN_MOAMR1" address="0xf000462c" entry="false"/>
-    <symbol name="CAN_MOAMR2" address="0xf000464c" entry="false"/>
-    <symbol name="CAN_MOAMR3" address="0xf000466c" entry="false"/>
-    <symbol name="CAN_MOAMR4" address="0xf000468c" entry="false"/>
-    <symbol name="CAN_MOAMR5" address="0xf00046ac" entry="false"/>
-    <symbol name="CAN_MOAMR6" address="0xf00046cc" entry="false"/>
-    <symbol name="CAN_MOAMR7" address="0xf00046ec" entry="false"/>
-    <symbol name="CAN_MOAMR8" address="0xf000470c" entry="false"/>
-    <symbol name="CAN_MOAMR9" address="0xf000472c" entry="false"/>
-    <symbol name="CAN_MOAMR10" address="0xf000474c" entry="false"/>
-    <symbol name="CAN_MOAMR11" address="0xf000476c" entry="false"/>
-    <symbol name="CAN_MOAMR12" address="0xf000478c" entry="false"/>
-    <symbol name="CAN_MOAMR13" address="0xf00047ac" entry="false"/>
-    <symbol name="CAN_MOAMR14" address="0xf00047cc" entry="false"/>
-    <symbol name="CAN_MOAMR15" address="0xf00047ec" entry="false"/>
-    <symbol name="CAN_MOAMR16" address="0xf000480c" entry="false"/>
-    <symbol name="CAN_MOAMR17" address="0xf000482c" entry="false"/>
-    <symbol name="CAN_MOAMR18" address="0xf000484c" entry="false"/>
-    <symbol name="CAN_MOAMR19" address="0xf000486c" entry="false"/>
-    <symbol name="CAN_MOAMR20" address="0xf000488c" entry="false"/>
-    <symbol name="CAN_MOAMR21" address="0xf00048ac" entry="false"/>
-    <symbol name="CAN_MOAMR22" address="0xf00048cc" entry="false"/>
-    <symbol name="CAN_MOAMR23" address="0xf00048ec" entry="false"/>
-    <symbol name="CAN_MOAMR24" address="0xf000490c" entry="false"/>
-    <symbol name="CAN_MOAMR25" address="0xf000492c" entry="false"/>
-    <symbol name="CAN_MOAMR26" address="0xf000494c" entry="false"/>
-    <symbol name="CAN_MOAMR27" address="0xf000496c" entry="false"/>
-    <symbol name="CAN_MOAMR28" address="0xf000498c" entry="false"/>
-    <symbol name="CAN_MOAMR29" address="0xf00049ac" entry="false"/>
-    <symbol name="CAN_MOAMR30" address="0xf00049cc" entry="false"/>
-    <symbol name="CAN_MOAMR31" address="0xf00049ec" entry="false"/>
-    <symbol name="CAN_MOAMR32" address="0xf0004a0c" entry="false"/>
-    <symbol name="CAN_MOAMR33" address="0xf0004a2c" entry="false"/>
-    <symbol name="CAN_MOAMR34" address="0xf0004a4c" entry="false"/>
-    <symbol name="CAN_MOAMR35" address="0xf0004a6c" entry="false"/>
-    <symbol name="CAN_MOAMR36" address="0xf0004a8c" entry="false"/>
-    <symbol name="CAN_MOAMR37" address="0xf0004aac" entry="false"/>
-    <symbol name="CAN_MOAMR38" address="0xf0004acc" entry="false"/>
-    <symbol name="CAN_MOAMR39" address="0xf0004aec" entry="false"/>
-    <symbol name="CAN_MOAMR40" address="0xf0004b0c" entry="false"/>
-    <symbol name="CAN_MOAMR41" address="0xf0004b2c" entry="false"/>
-    <symbol name="CAN_MOAMR42" address="0xf0004b4c" entry="false"/>
-    <symbol name="CAN_MOAMR43" address="0xf0004b6c" entry="false"/>
-    <symbol name="CAN_MOAMR44" address="0xf0004b8c" entry="false"/>
-    <symbol name="CAN_MOAMR45" address="0xf0004bac" entry="false"/>
-    <symbol name="CAN_MOAMR46" address="0xf0004bcc" entry="false"/>
-    <symbol name="CAN_MOAMR47" address="0xf0004bec" entry="false"/>
-    <symbol name="CAN_MOAMR48" address="0xf0004c0c" entry="false"/>
-    <symbol name="CAN_MOAMR49" address="0xf0004c2c" entry="false"/>
-    <symbol name="CAN_MOAMR50" address="0xf0004c4c" entry="false"/>
-    <symbol name="CAN_MOAMR51" address="0xf0004c6c" entry="false"/>
-    <symbol name="CAN_MOAMR52" address="0xf0004c8c" entry="false"/>
-    <symbol name="CAN_MOAMR53" address="0xf0004cac" entry="false"/>
-    <symbol name="CAN_MOAMR54" address="0xf0004ccc" entry="false"/>
-    <symbol name="CAN_MOAMR55" address="0xf0004cec" entry="false"/>
-    <symbol name="CAN_MOAMR56" address="0xf0004d0c" entry="false"/>
-    <symbol name="CAN_MOAMR57" address="0xf0004d2c" entry="false"/>
-    <symbol name="CAN_MOAMR58" address="0xf0004d4c" entry="false"/>
-    <symbol name="CAN_MOAMR59" address="0xf0004d6c" entry="false"/>
-    <symbol name="CAN_MOAMR60" address="0xf0004d8c" entry="false"/>
-    <symbol name="CAN_MOAMR61" address="0xf0004dac" entry="false"/>
-    <symbol name="CAN_MOAMR62" address="0xf0004dcc" entry="false"/>
-    <symbol name="CAN_MOAMR63" address="0xf0004dec" entry="false"/>
-    <symbol name="CAN_MODATAL0" address="0xf0004610" entry="false"/>
-    <symbol name="CAN_MODATAL1" address="0xf0004630" entry="false"/>
-    <symbol name="CAN_MODATAL2" address="0xf0004650" entry="false"/>
-    <symbol name="CAN_MODATAL3" address="0xf0004670" entry="false"/>
-    <symbol name="CAN_MODATAL4" address="0xf0004690" entry="false"/>
-    <symbol name="CAN_MODATAL5" address="0xf00046b0" entry="false"/>
-    <symbol name="CAN_MODATAL6" address="0xf00046d0" entry="false"/>
-    <symbol name="CAN_MODATAL7" address="0xf00046f0" entry="false"/>
-    <symbol name="CAN_MODATAL8" address="0xf0004710" entry="false"/>
-    <symbol name="CAN_MODATAL9" address="0xf0004730" entry="false"/>
-    <symbol name="CAN_MODATAL10" address="0xf0004750" entry="false"/>
-    <symbol name="CAN_MODATAL11" address="0xf0004770" entry="false"/>
-    <symbol name="CAN_MODATAL12" address="0xf0004790" entry="false"/>
-    <symbol name="CAN_MODATAL13" address="0xf00047b0" entry="false"/>
-    <symbol name="CAN_MODATAL14" address="0xf00047d0" entry="false"/>
-    <symbol name="CAN_MODATAL15" address="0xf00047f0" entry="false"/>
-    <symbol name="CAN_MODATAL16" address="0xf0004810" entry="false"/>
-    <symbol name="CAN_MODATAL17" address="0xf0004830" entry="false"/>
-    <symbol name="CAN_MODATAL18" address="0xf0004850" entry="false"/>
-    <symbol name="CAN_MODATAL19" address="0xf0004870" entry="false"/>
-    <symbol name="CAN_MODATAL20" address="0xf0004890" entry="false"/>
-    <symbol name="CAN_MODATAL21" address="0xf00048b0" entry="false"/>
-    <symbol name="CAN_MODATAL22" address="0xf00048d0" entry="false"/>
-    <symbol name="CAN_MODATAL23" address="0xf00048f0" entry="false"/>
-    <symbol name="CAN_MODATAL24" address="0xf0004910" entry="false"/>
-    <symbol name="CAN_MODATAL25" address="0xf0004930" entry="false"/>
-    <symbol name="CAN_MODATAL26" address="0xf0004950" entry="false"/>
-    <symbol name="CAN_MODATAL27" address="0xf0004970" entry="false"/>
-    <symbol name="CAN_MODATAL28" address="0xf0004990" entry="false"/>
-    <symbol name="CAN_MODATAL29" address="0xf00049b0" entry="false"/>
-    <symbol name="CAN_MODATAL30" address="0xf00049d0" entry="false"/>
-    <symbol name="CAN_MODATAL31" address="0xf00049f0" entry="false"/>
-    <symbol name="CAN_MODATAL32" address="0xf0004a10" entry="false"/>
-    <symbol name="CAN_MODATAL33" address="0xf0004a30" entry="false"/>
-    <symbol name="CAN_MODATAL34" address="0xf0004a50" entry="false"/>
-    <symbol name="CAN_MODATAL35" address="0xf0004a70" entry="false"/>
-    <symbol name="CAN_MODATAL36" address="0xf0004a90" entry="false"/>
-    <symbol name="CAN_MODATAL37" address="0xf0004ab0" entry="false"/>
-    <symbol name="CAN_MODATAL38" address="0xf0004ad0" entry="false"/>
-    <symbol name="CAN_MODATAL39" address="0xf0004af0" entry="false"/>
-    <symbol name="CAN_MODATAL40" address="0xf0004b10" entry="false"/>
-    <symbol name="CAN_MODATAL41" address="0xf0004b30" entry="false"/>
-    <symbol name="CAN_MODATAL42" address="0xf0004b50" entry="false"/>
-    <symbol name="CAN_MODATAL43" address="0xf0004b70" entry="false"/>
-    <symbol name="CAN_MODATAL44" address="0xf0004b90" entry="false"/>
-    <symbol name="CAN_MODATAL45" address="0xf0004bb0" entry="false"/>
-    <symbol name="CAN_MODATAL46" address="0xf0004bd0" entry="false"/>
-    <symbol name="CAN_MODATAL47" address="0xf0004bf0" entry="false"/>
-    <symbol name="CAN_MODATAL48" address="0xf0004c10" entry="false"/>
-    <symbol name="CAN_MODATAL49" address="0xf0004c30" entry="false"/>
-    <symbol name="CAN_MODATAL50" address="0xf0004c50" entry="false"/>
-    <symbol name="CAN_MODATAL51" address="0xf0004c70" entry="false"/>
-    <symbol name="CAN_MODATAL52" address="0xf0004c90" entry="false"/>
-    <symbol name="CAN_MODATAL53" address="0xf0004cb0" entry="false"/>
-    <symbol name="CAN_MODATAL54" address="0xf0004cd0" entry="false"/>
-    <symbol name="CAN_MODATAL55" address="0xf0004cf0" entry="false"/>
-    <symbol name="CAN_MODATAL56" address="0xf0004d10" entry="false"/>
-    <symbol name="CAN_MODATAL57" address="0xf0004d30" entry="false"/>
-    <symbol name="CAN_MODATAL58" address="0xf0004d50" entry="false"/>
-    <symbol name="CAN_MODATAL59" address="0xf0004d70" entry="false"/>
-    <symbol name="CAN_MODATAL60" address="0xf0004d90" entry="false"/>
-    <symbol name="CAN_MODATAL61" address="0xf0004db0" entry="false"/>
-    <symbol name="CAN_MODATAL62" address="0xf0004dd0" entry="false"/>
-    <symbol name="CAN_MODATAL63" address="0xf0004df0" entry="false"/>
-    <symbol name="CAN_MODATAH0" address="0xf0004614" entry="false"/>
-    <symbol name="CAN_MODATAH1" address="0xf0004634" entry="false"/>
-    <symbol name="CAN_MODATAH2" address="0xf0004654" entry="false"/>
-    <symbol name="CAN_MODATAH3" address="0xf0004674" entry="false"/>
-    <symbol name="CAN_MODATAH4" address="0xf0004694" entry="false"/>
-    <symbol name="CAN_MODATAH5" address="0xf00046b4" entry="false"/>
-    <symbol name="CAN_MODATAH6" address="0xf00046d4" entry="false"/>
-    <symbol name="CAN_MODATAH7" address="0xf00046f4" entry="false"/>
-    <symbol name="CAN_MODATAH8" address="0xf0004714" entry="false"/>
-    <symbol name="CAN_MODATAH9" address="0xf0004734" entry="false"/>
-    <symbol name="CAN_MODATAH10" address="0xf0004754" entry="false"/>
-    <symbol name="CAN_MODATAH11" address="0xf0004774" entry="false"/>
-    <symbol name="CAN_MODATAH12" address="0xf0004794" entry="false"/>
-    <symbol name="CAN_MODATAH13" address="0xf00047b4" entry="false"/>
-    <symbol name="CAN_MODATAH14" address="0xf00047d4" entry="false"/>
-    <symbol name="CAN_MODATAH15" address="0xf00047f4" entry="false"/>
-    <symbol name="CAN_MODATAH16" address="0xf0004814" entry="false"/>
-    <symbol name="CAN_MODATAH17" address="0xf0004834" entry="false"/>
-    <symbol name="CAN_MODATAH18" address="0xf0004854" entry="false"/>
-    <symbol name="CAN_MODATAH19" address="0xf0004874" entry="false"/>
-    <symbol name="CAN_MODATAH20" address="0xf0004894" entry="false"/>
-    <symbol name="CAN_MODATAH21" address="0xf00048b4" entry="false"/>
-    <symbol name="CAN_MODATAH22" address="0xf00048d4" entry="false"/>
-    <symbol name="CAN_MODATAH23" address="0xf00048f4" entry="false"/>
-    <symbol name="CAN_MODATAH24" address="0xf0004914" entry="false"/>
-    <symbol name="CAN_MODATAH25" address="0xf0004934" entry="false"/>
-    <symbol name="CAN_MODATAH26" address="0xf0004954" entry="false"/>
-    <symbol name="CAN_MODATAH27" address="0xf0004974" entry="false"/>
-    <symbol name="CAN_MODATAH28" address="0xf0004994" entry="false"/>
-    <symbol name="CAN_MODATAH29" address="0xf00049b4" entry="false"/>
-    <symbol name="CAN_MODATAH30" address="0xf00049d4" entry="false"/>
-    <symbol name="CAN_MODATAH31" address="0xf00049f4" entry="false"/>
-    <symbol name="CAN_MODATAH32" address="0xf0004a14" entry="false"/>
-    <symbol name="CAN_MODATAH33" address="0xf0004a34" entry="false"/>
-    <symbol name="CAN_MODATAH34" address="0xf0004a54" entry="false"/>
-    <symbol name="CAN_MODATAH35" address="0xf0004a74" entry="false"/>
-    <symbol name="CAN_MODATAH36" address="0xf0004a94" entry="false"/>
-    <symbol name="CAN_MODATAH37" address="0xf0004ab4" entry="false"/>
-    <symbol name="CAN_MODATAH38" address="0xf0004ad4" entry="false"/>
-    <symbol name="CAN_MODATAH39" address="0xf0004af4" entry="false"/>
-    <symbol name="CAN_MODATAH40" address="0xf0004b14" entry="false"/>
-    <symbol name="CAN_MODATAH41" address="0xf0004b34" entry="false"/>
-    <symbol name="CAN_MODATAH42" address="0xf0004b54" entry="false"/>
-    <symbol name="CAN_MODATAH43" address="0xf0004b74" entry="false"/>
-    <symbol name="CAN_MODATAH44" address="0xf0004b94" entry="false"/>
-    <symbol name="CAN_MODATAH45" address="0xf0004bb4" entry="false"/>
-    <symbol name="CAN_MODATAH46" address="0xf0004bd4" entry="false"/>
-    <symbol name="CAN_MODATAH47" address="0xf0004bf4" entry="false"/>
-    <symbol name="CAN_MODATAH48" address="0xf0004c14" entry="false"/>
-    <symbol name="CAN_MODATAH49" address="0xf0004c34" entry="false"/>
-    <symbol name="CAN_MODATAH50" address="0xf0004c54" entry="false"/>
-    <symbol name="CAN_MODATAH51" address="0xf0004c74" entry="false"/>
-    <symbol name="CAN_MODATAH52" address="0xf0004c94" entry="false"/>
-    <symbol name="CAN_MODATAH53" address="0xf0004cb4" entry="false"/>
-    <symbol name="CAN_MODATAH54" address="0xf0004cd4" entry="false"/>
-    <symbol name="CAN_MODATAH55" address="0xf0004cf4" entry="false"/>
-    <symbol name="CAN_MODATAH56" address="0xf0004d14" entry="false"/>
-    <symbol name="CAN_MODATAH57" address="0xf0004d34" entry="false"/>
-    <symbol name="CAN_MODATAH58" address="0xf0004d54" entry="false"/>
-    <symbol name="CAN_MODATAH59" address="0xf0004d74" entry="false"/>
-    <symbol name="CAN_MODATAH60" address="0xf0004d94" entry="false"/>
-    <symbol name="CAN_MODATAH61" address="0xf0004db4" entry="false"/>
-    <symbol name="CAN_MODATAH62" address="0xf0004dd4" entry="false"/>
-    <symbol name="CAN_MODATAH63" address="0xf0004df4" entry="false"/>
-    <symbol name="CAN_MOAR0" address="0xf0004618" entry="false"/>
-    <symbol name="CAN_MOAR1" address="0xf0004638" entry="false"/>
-    <symbol name="CAN_MOAR2" address="0xf0004658" entry="false"/>
-    <symbol name="CAN_MOAR3" address="0xf0004678" entry="false"/>
-    <symbol name="CAN_MOAR4" address="0xf0004698" entry="false"/>
-    <symbol name="CAN_MOAR5" address="0xf00046b8" entry="false"/>
-    <symbol name="CAN_MOAR6" address="0xf00046d8" entry="false"/>
-    <symbol name="CAN_MOAR7" address="0xf00046f8" entry="false"/>
-    <symbol name="CAN_MOAR8" address="0xf0004718" entry="false"/>
-    <symbol name="CAN_MOAR9" address="0xf0004738" entry="false"/>
-    <symbol name="CAN_MOAR10" address="0xf0004758" entry="false"/>
-    <symbol name="CAN_MOAR11" address="0xf0004778" entry="false"/>
-    <symbol name="CAN_MOAR12" address="0xf0004798" entry="false"/>
-    <symbol name="CAN_MOAR13" address="0xf00047b8" entry="false"/>
-    <symbol name="CAN_MOAR14" address="0xf00047d8" entry="false"/>
-    <symbol name="CAN_MOAR15" address="0xf00047f8" entry="false"/>
-    <symbol name="CAN_MOAR16" address="0xf0004818" entry="false"/>
-    <symbol name="CAN_MOAR17" address="0xf0004838" entry="false"/>
-    <symbol name="CAN_MOAR18" address="0xf0004858" entry="false"/>
-    <symbol name="CAN_MOAR19" address="0xf0004878" entry="false"/>
-    <symbol name="CAN_MOAR20" address="0xf0004898" entry="false"/>
-    <symbol name="CAN_MOAR21" address="0xf00048b8" entry="false"/>
-    <symbol name="CAN_MOAR22" address="0xf00048d8" entry="false"/>
-    <symbol name="CAN_MOAR23" address="0xf00048f8" entry="false"/>
-    <symbol name="CAN_MOAR24" address="0xf0004918" entry="false"/>
-    <symbol name="CAN_MOAR25" address="0xf0004938" entry="false"/>
-    <symbol name="CAN_MOAR26" address="0xf0004958" entry="false"/>
-    <symbol name="CAN_MOAR27" address="0xf0004978" entry="false"/>
-    <symbol name="CAN_MOAR28" address="0xf0004998" entry="false"/>
-    <symbol name="CAN_MOAR29" address="0xf00049b8" entry="false"/>
-    <symbol name="CAN_MOAR30" address="0xf00049d8" entry="false"/>
-    <symbol name="CAN_MOAR31" address="0xf00049f8" entry="false"/>
-    <symbol name="CAN_MOAR32" address="0xf0004a18" entry="false"/>
-    <symbol name="CAN_MOAR33" address="0xf0004a38" entry="false"/>
-    <symbol name="CAN_MOAR34" address="0xf0004a58" entry="false"/>
-    <symbol name="CAN_MOAR35" address="0xf0004a78" entry="false"/>
-    <symbol name="CAN_MOAR36" address="0xf0004a98" entry="false"/>
-    <symbol name="CAN_MOAR37" address="0xf0004ab8" entry="false"/>
-    <symbol name="CAN_MOAR38" address="0xf0004ad8" entry="false"/>
-    <symbol name="CAN_MOAR39" address="0xf0004af8" entry="false"/>
-    <symbol name="CAN_MOAR40" address="0xf0004b18" entry="false"/>
-    <symbol name="CAN_MOAR41" address="0xf0004b38" entry="false"/>
-    <symbol name="CAN_MOAR42" address="0xf0004b58" entry="false"/>
-    <symbol name="CAN_MOAR43" address="0xf0004b78" entry="false"/>
-    <symbol name="CAN_MOAR44" address="0xf0004b98" entry="false"/>
-    <symbol name="CAN_MOAR45" address="0xf0004bb8" entry="false"/>
-    <symbol name="CAN_MOAR46" address="0xf0004bd8" entry="false"/>
-    <symbol name="CAN_MOAR47" address="0xf0004bf8" entry="false"/>
-    <symbol name="CAN_MOAR48" address="0xf0004c18" entry="false"/>
-    <symbol name="CAN_MOAR49" address="0xf0004c38" entry="false"/>
-    <symbol name="CAN_MOAR50" address="0xf0004c58" entry="false"/>
-    <symbol name="CAN_MOAR51" address="0xf0004c78" entry="false"/>
-    <symbol name="CAN_MOAR52" address="0xf0004c98" entry="false"/>
-    <symbol name="CAN_MOAR53" address="0xf0004cb8" entry="false"/>
-    <symbol name="CAN_MOAR54" address="0xf0004cd8" entry="false"/>
-    <symbol name="CAN_MOAR55" address="0xf0004cf8" entry="false"/>
-    <symbol name="CAN_MOAR56" address="0xf0004d18" entry="false"/>
-    <symbol name="CAN_MOAR57" address="0xf0004d38" entry="false"/>
-    <symbol name="CAN_MOAR58" address="0xf0004d58" entry="false"/>
-    <symbol name="CAN_MOAR59" address="0xf0004d78" entry="false"/>
-    <symbol name="CAN_MOAR60" address="0xf0004d98" entry="false"/>
-    <symbol name="CAN_MOAR61" address="0xf0004db8" entry="false"/>
-    <symbol name="CAN_MOAR62" address="0xf0004dd8" entry="false"/>
-    <symbol name="CAN_MOAR63" address="0xf0004df8" entry="false"/>
-    <symbol name="CAN_MOCTR0" address="0xf000461c" entry="false"/>
-    <symbol name="CAN_MOCTR1" address="0xf000463c" entry="false"/>
-    <symbol name="CAN_MOCTR2" address="0xf000465c" entry="false"/>
-    <symbol name="CAN_MOCTR3" address="0xf000467c" entry="false"/>
-    <symbol name="CAN_MOCTR4" address="0xf000469c" entry="false"/>
-    <symbol name="CAN_MOCTR5" address="0xf00046bc" entry="false"/>
-    <symbol name="CAN_MOCTR6" address="0xf00046dc" entry="false"/>
-    <symbol name="CAN_MOCTR7" address="0xf00046fc" entry="false"/>
-    <symbol name="CAN_MOCTR8" address="0xf000471c" entry="false"/>
-    <symbol name="CAN_MOCTR9" address="0xf000473c" entry="false"/>
-    <symbol name="CAN_MOCTR10" address="0xf000475c" entry="false"/>
-    <symbol name="CAN_MOCTR11" address="0xf000477c" entry="false"/>
-    <symbol name="CAN_MOCTR12" address="0xf000479c" entry="false"/>
-    <symbol name="CAN_MOCTR13" address="0xf00047bc" entry="false"/>
-    <symbol name="CAN_MOCTR14" address="0xf00047dc" entry="false"/>
-    <symbol name="CAN_MOCTR15" address="0xf00047fc" entry="false"/>
-    <symbol name="CAN_MOCTR16" address="0xf000481c" entry="false"/>
-    <symbol name="CAN_MOCTR17" address="0xf000483c" entry="false"/>
-    <symbol name="CAN_MOCTR18" address="0xf000485c" entry="false"/>
-    <symbol name="CAN_MOCTR19" address="0xf000487c" entry="false"/>
-    <symbol name="CAN_MOCTR20" address="0xf000489c" entry="false"/>
-    <symbol name="CAN_MOCTR21" address="0xf00048bc" entry="false"/>
-    <symbol name="CAN_MOCTR22" address="0xf00048dc" entry="false"/>
-    <symbol name="CAN_MOCTR23" address="0xf00048fc" entry="false"/>
-    <symbol name="CAN_MOCTR24" address="0xf000491c" entry="false"/>
-    <symbol name="CAN_MOCTR25" address="0xf000493c" entry="false"/>
-    <symbol name="CAN_MOCTR26" address="0xf000495c" entry="false"/>
-    <symbol name="CAN_MOCTR27" address="0xf000497c" entry="false"/>
-    <symbol name="CAN_MOCTR28" address="0xf000499c" entry="false"/>
-    <symbol name="CAN_MOCTR29" address="0xf00049bc" entry="false"/>
-    <symbol name="CAN_MOCTR30" address="0xf00049dc" entry="false"/>
-    <symbol name="CAN_MOCTR31" address="0xf00049fc" entry="false"/>
-    <symbol name="CAN_MOCTR32" address="0xf0004a1c" entry="false"/>
-    <symbol name="CAN_MOCTR33" address="0xf0004a3c" entry="false"/>
-    <symbol name="CAN_MOCTR34" address="0xf0004a5c" entry="false"/>
-    <symbol name="CAN_MOCTR35" address="0xf0004a7c" entry="false"/>
-    <symbol name="CAN_MOCTR36" address="0xf0004a9c" entry="false"/>
-    <symbol name="CAN_MOCTR37" address="0xf0004abc" entry="false"/>
-    <symbol name="CAN_MOCTR38" address="0xf0004adc" entry="false"/>
-    <symbol name="CAN_MOCTR39" address="0xf0004afc" entry="false"/>
-    <symbol name="CAN_MOCTR40" address="0xf0004b1c" entry="false"/>
-    <symbol name="CAN_MOCTR41" address="0xf0004b3c" entry="false"/>
-    <symbol name="CAN_MOCTR42" address="0xf0004b5c" entry="false"/>
-    <symbol name="CAN_MOCTR43" address="0xf0004b7c" entry="false"/>
-    <symbol name="CAN_MOCTR44" address="0xf0004b9c" entry="false"/>
-    <symbol name="CAN_MOCTR45" address="0xf0004bbc" entry="false"/>
-    <symbol name="CAN_MOCTR46" address="0xf0004bdc" entry="false"/>
-    <symbol name="CAN_MOCTR47" address="0xf0004bfc" entry="false"/>
-    <symbol name="CAN_MOCTR48" address="0xf0004c1c" entry="false"/>
-    <symbol name="CAN_MOCTR49" address="0xf0004c3c" entry="false"/>
-    <symbol name="CAN_MOCTR50" address="0xf0004c5c" entry="false"/>
-    <symbol name="CAN_MOCTR51" address="0xf0004c7c" entry="false"/>
-    <symbol name="CAN_MOCTR52" address="0xf0004c9c" entry="false"/>
-    <symbol name="CAN_MOCTR53" address="0xf0004cbc" entry="false"/>
-    <symbol name="CAN_MOCTR54" address="0xf0004cdc" entry="false"/>
-    <symbol name="CAN_MOCTR55" address="0xf0004cfc" entry="false"/>
-    <symbol name="CAN_MOCTR56" address="0xf0004d1c" entry="false"/>
-    <symbol name="CAN_MOCTR57" address="0xf0004d3c" entry="false"/>
-    <symbol name="CAN_MOCTR58" address="0xf0004d5c" entry="false"/>
-    <symbol name="CAN_MOCTR59" address="0xf0004d7c" entry="false"/>
-    <symbol name="CAN_MOCTR60" address="0xf0004d9c" entry="false"/>
-    <symbol name="CAN_MOCTR61" address="0xf0004dbc" entry="false"/>
-    <symbol name="CAN_MOCTR62" address="0xf0004ddc" entry="false"/>
-    <symbol name="CAN_MOCTR63" address="0xf0004dfc" entry="false"/>
+    <symbol name="CAN_MOFCR0" address="0xf0004400" entry="false"/>
+    <symbol name="CAN_MOFCR1" address="0xf0004420" entry="false"/>
+    <symbol name="CAN_MOFCR2" address="0xf0004440" entry="false"/>
+    <symbol name="CAN_MOFCR3" address="0xf0004460" entry="false"/>
+    <symbol name="CAN_MOFCR4" address="0xf0004480" entry="false"/>
+    <symbol name="CAN_MOFCR5" address="0xf00044a0" entry="false"/>
+    <symbol name="CAN_MOFCR6" address="0xf00044c0" entry="false"/>
+    <symbol name="CAN_MOFCR7" address="0xf00044e0" entry="false"/>
+    <symbol name="CAN_MOFCR8" address="0xf0004500" entry="false"/>
+    <symbol name="CAN_MOFCR9" address="0xf0004520" entry="false"/>
+    <symbol name="CAN_MOFCR10" address="0xf0004540" entry="false"/>
+    <symbol name="CAN_MOFCR11" address="0xf0004560" entry="false"/>
+    <symbol name="CAN_MOFCR12" address="0xf0004580" entry="false"/>
+    <symbol name="CAN_MOFCR13" address="0xf00045a0" entry="false"/>
+    <symbol name="CAN_MOFCR14" address="0xf00045c0" entry="false"/>
+    <symbol name="CAN_MOFCR15" address="0xf00045e0" entry="false"/>
+    <symbol name="CAN_MOFCR16" address="0xf0004600" entry="false"/>
+    <symbol name="CAN_MOFCR17" address="0xf0004620" entry="false"/>
+    <symbol name="CAN_MOFCR18" address="0xf0004640" entry="false"/>
+    <symbol name="CAN_MOFCR19" address="0xf0004660" entry="false"/>
+    <symbol name="CAN_MOFCR20" address="0xf0004680" entry="false"/>
+    <symbol name="CAN_MOFCR21" address="0xf00046a0" entry="false"/>
+    <symbol name="CAN_MOFCR22" address="0xf00046c0" entry="false"/>
+    <symbol name="CAN_MOFCR23" address="0xf00046e0" entry="false"/>
+    <symbol name="CAN_MOFCR24" address="0xf0004700" entry="false"/>
+    <symbol name="CAN_MOFCR25" address="0xf0004720" entry="false"/>
+    <symbol name="CAN_MOFCR26" address="0xf0004740" entry="false"/>
+    <symbol name="CAN_MOFCR27" address="0xf0004760" entry="false"/>
+    <symbol name="CAN_MOFCR28" address="0xf0004780" entry="false"/>
+    <symbol name="CAN_MOFCR29" address="0xf00047a0" entry="false"/>
+    <symbol name="CAN_MOFCR30" address="0xf00047c0" entry="false"/>
+    <symbol name="CAN_MOFCR31" address="0xf00047e0" entry="false"/>
+    <symbol name="CAN_MOFCR32" address="0xf0004800" entry="false"/>
+    <symbol name="CAN_MOFCR33" address="0xf0004820" entry="false"/>
+    <symbol name="CAN_MOFCR34" address="0xf0004840" entry="false"/>
+    <symbol name="CAN_MOFCR35" address="0xf0004860" entry="false"/>
+    <symbol name="CAN_MOFCR36" address="0xf0004880" entry="false"/>
+    <symbol name="CAN_MOFCR37" address="0xf00048a0" entry="false"/>
+    <symbol name="CAN_MOFCR38" address="0xf00048c0" entry="false"/>
+    <symbol name="CAN_MOFCR39" address="0xf00048e0" entry="false"/>
+    <symbol name="CAN_MOFCR40" address="0xf0004900" entry="false"/>
+    <symbol name="CAN_MOFCR41" address="0xf0004920" entry="false"/>
+    <symbol name="CAN_MOFCR42" address="0xf0004940" entry="false"/>
+    <symbol name="CAN_MOFCR43" address="0xf0004960" entry="false"/>
+    <symbol name="CAN_MOFCR44" address="0xf0004980" entry="false"/>
+    <symbol name="CAN_MOFCR45" address="0xf00049a0" entry="false"/>
+    <symbol name="CAN_MOFCR46" address="0xf00049c0" entry="false"/>
+    <symbol name="CAN_MOFCR47" address="0xf00049e0" entry="false"/>
+    <symbol name="CAN_MOFCR48" address="0xf0004a00" entry="false"/>
+    <symbol name="CAN_MOFCR49" address="0xf0004a20" entry="false"/>
+    <symbol name="CAN_MOFCR50" address="0xf0004a40" entry="false"/>
+    <symbol name="CAN_MOFCR51" address="0xf0004a60" entry="false"/>
+    <symbol name="CAN_MOFCR52" address="0xf0004a80" entry="false"/>
+    <symbol name="CAN_MOFCR53" address="0xf0004aa0" entry="false"/>
+    <symbol name="CAN_MOFCR54" address="0xf0004ac0" entry="false"/>
+    <symbol name="CAN_MOFCR55" address="0xf0004ae0" entry="false"/>
+    <symbol name="CAN_MOFCR56" address="0xf0004b00" entry="false"/>
+    <symbol name="CAN_MOFCR57" address="0xf0004b20" entry="false"/>
+    <symbol name="CAN_MOFCR58" address="0xf0004b40" entry="false"/>
+    <symbol name="CAN_MOFCR59" address="0xf0004b60" entry="false"/>
+    <symbol name="CAN_MOFCR60" address="0xf0004b80" entry="false"/>
+    <symbol name="CAN_MOFCR61" address="0xf0004ba0" entry="false"/>
+    <symbol name="CAN_MOFCR62" address="0xf0004bc0" entry="false"/>
+    <symbol name="CAN_MOFCR63" address="0xf0004be0" entry="false"/>
+    <symbol name="CAN_MOFGPR0" address="0xf0004404" entry="false"/>
+    <symbol name="CAN_MOFGPR1" address="0xf0004424" entry="false"/>
+    <symbol name="CAN_MOFGPR2" address="0xf0004444" entry="false"/>
+    <symbol name="CAN_MOFGPR3" address="0xf0004464" entry="false"/>
+    <symbol name="CAN_MOFGPR4" address="0xf0004484" entry="false"/>
+    <symbol name="CAN_MOFGPR5" address="0xf00044a4" entry="false"/>
+    <symbol name="CAN_MOFGPR6" address="0xf00044c4" entry="false"/>
+    <symbol name="CAN_MOFGPR7" address="0xf00044e4" entry="false"/>
+    <symbol name="CAN_MOFGPR8" address="0xf0004504" entry="false"/>
+    <symbol name="CAN_MOFGPR9" address="0xf0004524" entry="false"/>
+    <symbol name="CAN_MOFGPR10" address="0xf0004544" entry="false"/>
+    <symbol name="CAN_MOFGPR11" address="0xf0004564" entry="false"/>
+    <symbol name="CAN_MOFGPR12" address="0xf0004584" entry="false"/>
+    <symbol name="CAN_MOFGPR13" address="0xf00045a4" entry="false"/>
+    <symbol name="CAN_MOFGPR14" address="0xf00045c4" entry="false"/>
+    <symbol name="CAN_MOFGPR15" address="0xf00045e4" entry="false"/>
+    <symbol name="CAN_MOFGPR16" address="0xf0004604" entry="false"/>
+    <symbol name="CAN_MOFGPR17" address="0xf0004624" entry="false"/>
+    <symbol name="CAN_MOFGPR18" address="0xf0004644" entry="false"/>
+    <symbol name="CAN_MOFGPR19" address="0xf0004664" entry="false"/>
+    <symbol name="CAN_MOFGPR20" address="0xf0004684" entry="false"/>
+    <symbol name="CAN_MOFGPR21" address="0xf00046a4" entry="false"/>
+    <symbol name="CAN_MOFGPR22" address="0xf00046c4" entry="false"/>
+    <symbol name="CAN_MOFGPR23" address="0xf00046e4" entry="false"/>
+    <symbol name="CAN_MOFGPR24" address="0xf0004704" entry="false"/>
+    <symbol name="CAN_MOFGPR25" address="0xf0004724" entry="false"/>
+    <symbol name="CAN_MOFGPR26" address="0xf0004744" entry="false"/>
+    <symbol name="CAN_MOFGPR27" address="0xf0004764" entry="false"/>
+    <symbol name="CAN_MOFGPR28" address="0xf0004784" entry="false"/>
+    <symbol name="CAN_MOFGPR29" address="0xf00047a4" entry="false"/>
+    <symbol name="CAN_MOFGPR30" address="0xf00047c4" entry="false"/>
+    <symbol name="CAN_MOFGPR31" address="0xf00047e4" entry="false"/>
+    <symbol name="CAN_MOFGPR32" address="0xf0004804" entry="false"/>
+    <symbol name="CAN_MOFGPR33" address="0xf0004824" entry="false"/>
+    <symbol name="CAN_MOFGPR34" address="0xf0004844" entry="false"/>
+    <symbol name="CAN_MOFGPR35" address="0xf0004864" entry="false"/>
+    <symbol name="CAN_MOFGPR36" address="0xf0004884" entry="false"/>
+    <symbol name="CAN_MOFGPR37" address="0xf00048a4" entry="false"/>
+    <symbol name="CAN_MOFGPR38" address="0xf00048c4" entry="false"/>
+    <symbol name="CAN_MOFGPR39" address="0xf00048e4" entry="false"/>
+    <symbol name="CAN_MOFGPR40" address="0xf0004904" entry="false"/>
+    <symbol name="CAN_MOFGPR41" address="0xf0004924" entry="false"/>
+    <symbol name="CAN_MOFGPR42" address="0xf0004944" entry="false"/>
+    <symbol name="CAN_MOFGPR43" address="0xf0004964" entry="false"/>
+    <symbol name="CAN_MOFGPR44" address="0xf0004984" entry="false"/>
+    <symbol name="CAN_MOFGPR45" address="0xf00049a4" entry="false"/>
+    <symbol name="CAN_MOFGPR46" address="0xf00049c4" entry="false"/>
+    <symbol name="CAN_MOFGPR47" address="0xf00049e4" entry="false"/>
+    <symbol name="CAN_MOFGPR48" address="0xf0004a04" entry="false"/>
+    <symbol name="CAN_MOFGPR49" address="0xf0004a24" entry="false"/>
+    <symbol name="CAN_MOFGPR50" address="0xf0004a44" entry="false"/>
+    <symbol name="CAN_MOFGPR51" address="0xf0004a64" entry="false"/>
+    <symbol name="CAN_MOFGPR52" address="0xf0004a84" entry="false"/>
+    <symbol name="CAN_MOFGPR53" address="0xf0004aa4" entry="false"/>
+    <symbol name="CAN_MOFGPR54" address="0xf0004ac4" entry="false"/>
+    <symbol name="CAN_MOFGPR55" address="0xf0004ae4" entry="false"/>
+    <symbol name="CAN_MOFGPR56" address="0xf0004b04" entry="false"/>
+    <symbol name="CAN_MOFGPR57" address="0xf0004b24" entry="false"/>
+    <symbol name="CAN_MOFGPR58" address="0xf0004b44" entry="false"/>
+    <symbol name="CAN_MOFGPR59" address="0xf0004b64" entry="false"/>
+    <symbol name="CAN_MOFGPR60" address="0xf0004b84" entry="false"/>
+    <symbol name="CAN_MOFGPR61" address="0xf0004ba4" entry="false"/>
+    <symbol name="CAN_MOFGPR62" address="0xf0004bc4" entry="false"/>
+    <symbol name="CAN_MOFGPR63" address="0xf0004be4" entry="false"/>
+    <symbol name="CAN_MOIPR0" address="0xf0004408" entry="false"/>
+    <symbol name="CAN_MOIPR1" address="0xf0004428" entry="false"/>
+    <symbol name="CAN_MOIPR2" address="0xf0004448" entry="false"/>
+    <symbol name="CAN_MOIPR3" address="0xf0004468" entry="false"/>
+    <symbol name="CAN_MOIPR4" address="0xf0004488" entry="false"/>
+    <symbol name="CAN_MOIPR5" address="0xf00044a8" entry="false"/>
+    <symbol name="CAN_MOIPR6" address="0xf00044c8" entry="false"/>
+    <symbol name="CAN_MOIPR7" address="0xf00044e8" entry="false"/>
+    <symbol name="CAN_MOIPR8" address="0xf0004508" entry="false"/>
+    <symbol name="CAN_MOIPR9" address="0xf0004528" entry="false"/>
+    <symbol name="CAN_MOIPR10" address="0xf0004548" entry="false"/>
+    <symbol name="CAN_MOIPR11" address="0xf0004568" entry="false"/>
+    <symbol name="CAN_MOIPR12" address="0xf0004588" entry="false"/>
+    <symbol name="CAN_MOIPR13" address="0xf00045a8" entry="false"/>
+    <symbol name="CAN_MOIPR14" address="0xf00045c8" entry="false"/>
+    <symbol name="CAN_MOIPR15" address="0xf00045e8" entry="false"/>
+    <symbol name="CAN_MOIPR16" address="0xf0004608" entry="false"/>
+    <symbol name="CAN_MOIPR17" address="0xf0004628" entry="false"/>
+    <symbol name="CAN_MOIPR18" address="0xf0004648" entry="false"/>
+    <symbol name="CAN_MOIPR19" address="0xf0004668" entry="false"/>
+    <symbol name="CAN_MOIPR20" address="0xf0004688" entry="false"/>
+    <symbol name="CAN_MOIPR21" address="0xf00046a8" entry="false"/>
+    <symbol name="CAN_MOIPR22" address="0xf00046c8" entry="false"/>
+    <symbol name="CAN_MOIPR23" address="0xf00046e8" entry="false"/>
+    <symbol name="CAN_MOIPR24" address="0xf0004708" entry="false"/>
+    <symbol name="CAN_MOIPR25" address="0xf0004728" entry="false"/>
+    <symbol name="CAN_MOIPR26" address="0xf0004748" entry="false"/>
+    <symbol name="CAN_MOIPR27" address="0xf0004768" entry="false"/>
+    <symbol name="CAN_MOIPR28" address="0xf0004788" entry="false"/>
+    <symbol name="CAN_MOIPR29" address="0xf00047a8" entry="false"/>
+    <symbol name="CAN_MOIPR30" address="0xf00047c8" entry="false"/>
+    <symbol name="CAN_MOIPR31" address="0xf00047e8" entry="false"/>
+    <symbol name="CAN_MOIPR32" address="0xf0004808" entry="false"/>
+    <symbol name="CAN_MOIPR33" address="0xf0004828" entry="false"/>
+    <symbol name="CAN_MOIPR34" address="0xf0004848" entry="false"/>
+    <symbol name="CAN_MOIPR35" address="0xf0004868" entry="false"/>
+    <symbol name="CAN_MOIPR36" address="0xf0004888" entry="false"/>
+    <symbol name="CAN_MOIPR37" address="0xf00048a8" entry="false"/>
+    <symbol name="CAN_MOIPR38" address="0xf00048c8" entry="false"/>
+    <symbol name="CAN_MOIPR39" address="0xf00048e8" entry="false"/>
+    <symbol name="CAN_MOIPR40" address="0xf0004908" entry="false"/>
+    <symbol name="CAN_MOIPR41" address="0xf0004928" entry="false"/>
+    <symbol name="CAN_MOIPR42" address="0xf0004948" entry="false"/>
+    <symbol name="CAN_MOIPR43" address="0xf0004968" entry="false"/>
+    <symbol name="CAN_MOIPR44" address="0xf0004988" entry="false"/>
+    <symbol name="CAN_MOIPR45" address="0xf00049a8" entry="false"/>
+    <symbol name="CAN_MOIPR46" address="0xf00049c8" entry="false"/>
+    <symbol name="CAN_MOIPR47" address="0xf00049e8" entry="false"/>
+    <symbol name="CAN_MOIPR48" address="0xf0004a08" entry="false"/>
+    <symbol name="CAN_MOIPR49" address="0xf0004a28" entry="false"/>
+    <symbol name="CAN_MOIPR50" address="0xf0004a48" entry="false"/>
+    <symbol name="CAN_MOIPR51" address="0xf0004a68" entry="false"/>
+    <symbol name="CAN_MOIPR52" address="0xf0004a88" entry="false"/>
+    <symbol name="CAN_MOIPR53" address="0xf0004aa8" entry="false"/>
+    <symbol name="CAN_MOIPR54" address="0xf0004ac8" entry="false"/>
+    <symbol name="CAN_MOIPR55" address="0xf0004ae8" entry="false"/>
+    <symbol name="CAN_MOIPR56" address="0xf0004b08" entry="false"/>
+    <symbol name="CAN_MOIPR57" address="0xf0004b28" entry="false"/>
+    <symbol name="CAN_MOIPR58" address="0xf0004b48" entry="false"/>
+    <symbol name="CAN_MOIPR59" address="0xf0004b68" entry="false"/>
+    <symbol name="CAN_MOIPR60" address="0xf0004b88" entry="false"/>
+    <symbol name="CAN_MOIPR61" address="0xf0004ba8" entry="false"/>
+    <symbol name="CAN_MOIPR62" address="0xf0004bc8" entry="false"/>
+    <symbol name="CAN_MOIPR63" address="0xf0004be8" entry="false"/>
+    <symbol name="CAN_MOAMR0" address="0xf000440c" entry="false"/>
+    <symbol name="CAN_MOAMR1" address="0xf000442c" entry="false"/>
+    <symbol name="CAN_MOAMR2" address="0xf000444c" entry="false"/>
+    <symbol name="CAN_MOAMR3" address="0xf000446c" entry="false"/>
+    <symbol name="CAN_MOAMR4" address="0xf000448c" entry="false"/>
+    <symbol name="CAN_MOAMR5" address="0xf00044ac" entry="false"/>
+    <symbol name="CAN_MOAMR6" address="0xf00044cc" entry="false"/>
+    <symbol name="CAN_MOAMR7" address="0xf00044ec" entry="false"/>
+    <symbol name="CAN_MOAMR8" address="0xf000450c" entry="false"/>
+    <symbol name="CAN_MOAMR9" address="0xf000452c" entry="false"/>
+    <symbol name="CAN_MOAMR10" address="0xf000454c" entry="false"/>
+    <symbol name="CAN_MOAMR11" address="0xf000456c" entry="false"/>
+    <symbol name="CAN_MOAMR12" address="0xf000458c" entry="false"/>
+    <symbol name="CAN_MOAMR13" address="0xf00045ac" entry="false"/>
+    <symbol name="CAN_MOAMR14" address="0xf00045cc" entry="false"/>
+    <symbol name="CAN_MOAMR15" address="0xf00045ec" entry="false"/>
+    <symbol name="CAN_MOAMR16" address="0xf000460c" entry="false"/>
+    <symbol name="CAN_MOAMR17" address="0xf000462c" entry="false"/>
+    <symbol name="CAN_MOAMR18" address="0xf000464c" entry="false"/>
+    <symbol name="CAN_MOAMR19" address="0xf000466c" entry="false"/>
+    <symbol name="CAN_MOAMR20" address="0xf000468c" entry="false"/>
+    <symbol name="CAN_MOAMR21" address="0xf00046ac" entry="false"/>
+    <symbol name="CAN_MOAMR22" address="0xf00046cc" entry="false"/>
+    <symbol name="CAN_MOAMR23" address="0xf00046ec" entry="false"/>
+    <symbol name="CAN_MOAMR24" address="0xf000470c" entry="false"/>
+    <symbol name="CAN_MOAMR25" address="0xf000472c" entry="false"/>
+    <symbol name="CAN_MOAMR26" address="0xf000474c" entry="false"/>
+    <symbol name="CAN_MOAMR27" address="0xf000476c" entry="false"/>
+    <symbol name="CAN_MOAMR28" address="0xf000478c" entry="false"/>
+    <symbol name="CAN_MOAMR29" address="0xf00047ac" entry="false"/>
+    <symbol name="CAN_MOAMR30" address="0xf00047cc" entry="false"/>
+    <symbol name="CAN_MOAMR31" address="0xf00047ec" entry="false"/>
+    <symbol name="CAN_MOAMR32" address="0xf000480c" entry="false"/>
+    <symbol name="CAN_MOAMR33" address="0xf000482c" entry="false"/>
+    <symbol name="CAN_MOAMR34" address="0xf000484c" entry="false"/>
+    <symbol name="CAN_MOAMR35" address="0xf000486c" entry="false"/>
+    <symbol name="CAN_MOAMR36" address="0xf000488c" entry="false"/>
+    <symbol name="CAN_MOAMR37" address="0xf00048ac" entry="false"/>
+    <symbol name="CAN_MOAMR38" address="0xf00048cc" entry="false"/>
+    <symbol name="CAN_MOAMR39" address="0xf00048ec" entry="false"/>
+    <symbol name="CAN_MOAMR40" address="0xf000490c" entry="false"/>
+    <symbol name="CAN_MOAMR41" address="0xf000492c" entry="false"/>
+    <symbol name="CAN_MOAMR42" address="0xf000494c" entry="false"/>
+    <symbol name="CAN_MOAMR43" address="0xf000496c" entry="false"/>
+    <symbol name="CAN_MOAMR44" address="0xf000498c" entry="false"/>
+    <symbol name="CAN_MOAMR45" address="0xf00049ac" entry="false"/>
+    <symbol name="CAN_MOAMR46" address="0xf00049cc" entry="false"/>
+    <symbol name="CAN_MOAMR47" address="0xf00049ec" entry="false"/>
+    <symbol name="CAN_MOAMR48" address="0xf0004a0c" entry="false"/>
+    <symbol name="CAN_MOAMR49" address="0xf0004a2c" entry="false"/>
+    <symbol name="CAN_MOAMR50" address="0xf0004a4c" entry="false"/>
+    <symbol name="CAN_MOAMR51" address="0xf0004a6c" entry="false"/>
+    <symbol name="CAN_MOAMR52" address="0xf0004a8c" entry="false"/>
+    <symbol name="CAN_MOAMR53" address="0xf0004aac" entry="false"/>
+    <symbol name="CAN_MOAMR54" address="0xf0004acc" entry="false"/>
+    <symbol name="CAN_MOAMR55" address="0xf0004aec" entry="false"/>
+    <symbol name="CAN_MOAMR56" address="0xf0004b0c" entry="false"/>
+    <symbol name="CAN_MOAMR57" address="0xf0004b2c" entry="false"/>
+    <symbol name="CAN_MOAMR58" address="0xf0004b4c" entry="false"/>
+    <symbol name="CAN_MOAMR59" address="0xf0004b6c" entry="false"/>
+    <symbol name="CAN_MOAMR60" address="0xf0004b8c" entry="false"/>
+    <symbol name="CAN_MOAMR61" address="0xf0004bac" entry="false"/>
+    <symbol name="CAN_MOAMR62" address="0xf0004bcc" entry="false"/>
+    <symbol name="CAN_MOAMR63" address="0xf0004bec" entry="false"/>
+    <symbol name="CAN_MODATAL0" address="0xf0004410" entry="false"/>
+    <symbol name="CAN_MODATAL1" address="0xf0004430" entry="false"/>
+    <symbol name="CAN_MODATAL2" address="0xf0004450" entry="false"/>
+    <symbol name="CAN_MODATAL3" address="0xf0004470" entry="false"/>
+    <symbol name="CAN_MODATAL4" address="0xf0004490" entry="false"/>
+    <symbol name="CAN_MODATAL5" address="0xf00044b0" entry="false"/>
+    <symbol name="CAN_MODATAL6" address="0xf00044d0" entry="false"/>
+    <symbol name="CAN_MODATAL7" address="0xf00044f0" entry="false"/>
+    <symbol name="CAN_MODATAL8" address="0xf0004510" entry="false"/>
+    <symbol name="CAN_MODATAL9" address="0xf0004530" entry="false"/>
+    <symbol name="CAN_MODATAL10" address="0xf0004550" entry="false"/>
+    <symbol name="CAN_MODATAL11" address="0xf0004570" entry="false"/>
+    <symbol name="CAN_MODATAL12" address="0xf0004590" entry="false"/>
+    <symbol name="CAN_MODATAL13" address="0xf00045b0" entry="false"/>
+    <symbol name="CAN_MODATAL14" address="0xf00045d0" entry="false"/>
+    <symbol name="CAN_MODATAL15" address="0xf00045f0" entry="false"/>
+    <symbol name="CAN_MODATAL16" address="0xf0004610" entry="false"/>
+    <symbol name="CAN_MODATAL17" address="0xf0004630" entry="false"/>
+    <symbol name="CAN_MODATAL18" address="0xf0004650" entry="false"/>
+    <symbol name="CAN_MODATAL19" address="0xf0004670" entry="false"/>
+    <symbol name="CAN_MODATAL20" address="0xf0004690" entry="false"/>
+    <symbol name="CAN_MODATAL21" address="0xf00046b0" entry="false"/>
+    <symbol name="CAN_MODATAL22" address="0xf00046d0" entry="false"/>
+    <symbol name="CAN_MODATAL23" address="0xf00046f0" entry="false"/>
+    <symbol name="CAN_MODATAL24" address="0xf0004710" entry="false"/>
+    <symbol name="CAN_MODATAL25" address="0xf0004730" entry="false"/>
+    <symbol name="CAN_MODATAL26" address="0xf0004750" entry="false"/>
+    <symbol name="CAN_MODATAL27" address="0xf0004770" entry="false"/>
+    <symbol name="CAN_MODATAL28" address="0xf0004790" entry="false"/>
+    <symbol name="CAN_MODATAL29" address="0xf00047b0" entry="false"/>
+    <symbol name="CAN_MODATAL30" address="0xf00047d0" entry="false"/>
+    <symbol name="CAN_MODATAL31" address="0xf00047f0" entry="false"/>
+    <symbol name="CAN_MODATAL32" address="0xf0004810" entry="false"/>
+    <symbol name="CAN_MODATAL33" address="0xf0004830" entry="false"/>
+    <symbol name="CAN_MODATAL34" address="0xf0004850" entry="false"/>
+    <symbol name="CAN_MODATAL35" address="0xf0004870" entry="false"/>
+    <symbol name="CAN_MODATAL36" address="0xf0004890" entry="false"/>
+    <symbol name="CAN_MODATAL37" address="0xf00048b0" entry="false"/>
+    <symbol name="CAN_MODATAL38" address="0xf00048d0" entry="false"/>
+    <symbol name="CAN_MODATAL39" address="0xf00048f0" entry="false"/>
+    <symbol name="CAN_MODATAL40" address="0xf0004910" entry="false"/>
+    <symbol name="CAN_MODATAL41" address="0xf0004930" entry="false"/>
+    <symbol name="CAN_MODATAL42" address="0xf0004950" entry="false"/>
+    <symbol name="CAN_MODATAL43" address="0xf0004970" entry="false"/>
+    <symbol name="CAN_MODATAL44" address="0xf0004990" entry="false"/>
+    <symbol name="CAN_MODATAL45" address="0xf00049b0" entry="false"/>
+    <symbol name="CAN_MODATAL46" address="0xf00049d0" entry="false"/>
+    <symbol name="CAN_MODATAL47" address="0xf00049f0" entry="false"/>
+    <symbol name="CAN_MODATAL48" address="0xf0004a10" entry="false"/>
+    <symbol name="CAN_MODATAL49" address="0xf0004a30" entry="false"/>
+    <symbol name="CAN_MODATAL50" address="0xf0004a50" entry="false"/>
+    <symbol name="CAN_MODATAL51" address="0xf0004a70" entry="false"/>
+    <symbol name="CAN_MODATAL52" address="0xf0004a90" entry="false"/>
+    <symbol name="CAN_MODATAL53" address="0xf0004ab0" entry="false"/>
+    <symbol name="CAN_MODATAL54" address="0xf0004ad0" entry="false"/>
+    <symbol name="CAN_MODATAL55" address="0xf0004af0" entry="false"/>
+    <symbol name="CAN_MODATAL56" address="0xf0004b10" entry="false"/>
+    <symbol name="CAN_MODATAL57" address="0xf0004b30" entry="false"/>
+    <symbol name="CAN_MODATAL58" address="0xf0004b50" entry="false"/>
+    <symbol name="CAN_MODATAL59" address="0xf0004b70" entry="false"/>
+    <symbol name="CAN_MODATAL60" address="0xf0004b90" entry="false"/>
+    <symbol name="CAN_MODATAL61" address="0xf0004bb0" entry="false"/>
+    <symbol name="CAN_MODATAL62" address="0xf0004bd0" entry="false"/>
+    <symbol name="CAN_MODATAL63" address="0xf0004bf0" entry="false"/>
+    <symbol name="CAN_MODATAH0" address="0xf0004414" entry="false"/>
+    <symbol name="CAN_MODATAH1" address="0xf0004434" entry="false"/>
+    <symbol name="CAN_MODATAH2" address="0xf0004454" entry="false"/>
+    <symbol name="CAN_MODATAH3" address="0xf0004474" entry="false"/>
+    <symbol name="CAN_MODATAH4" address="0xf0004494" entry="false"/>
+    <symbol name="CAN_MODATAH5" address="0xf00044b4" entry="false"/>
+    <symbol name="CAN_MODATAH6" address="0xf00044d4" entry="false"/>
+    <symbol name="CAN_MODATAH7" address="0xf00044f4" entry="false"/>
+    <symbol name="CAN_MODATAH8" address="0xf0004514" entry="false"/>
+    <symbol name="CAN_MODATAH9" address="0xf0004534" entry="false"/>
+    <symbol name="CAN_MODATAH10" address="0xf0004554" entry="false"/>
+    <symbol name="CAN_MODATAH11" address="0xf0004574" entry="false"/>
+    <symbol name="CAN_MODATAH12" address="0xf0004594" entry="false"/>
+    <symbol name="CAN_MODATAH13" address="0xf00045b4" entry="false"/>
+    <symbol name="CAN_MODATAH14" address="0xf00045d4" entry="false"/>
+    <symbol name="CAN_MODATAH15" address="0xf00045f4" entry="false"/>
+    <symbol name="CAN_MODATAH16" address="0xf0004614" entry="false"/>
+    <symbol name="CAN_MODATAH17" address="0xf0004634" entry="false"/>
+    <symbol name="CAN_MODATAH18" address="0xf0004654" entry="false"/>
+    <symbol name="CAN_MODATAH19" address="0xf0004674" entry="false"/>
+    <symbol name="CAN_MODATAH20" address="0xf0004694" entry="false"/>
+    <symbol name="CAN_MODATAH21" address="0xf00046b4" entry="false"/>
+    <symbol name="CAN_MODATAH22" address="0xf00046d4" entry="false"/>
+    <symbol name="CAN_MODATAH23" address="0xf00046f4" entry="false"/>
+    <symbol name="CAN_MODATAH24" address="0xf0004714" entry="false"/>
+    <symbol name="CAN_MODATAH25" address="0xf0004734" entry="false"/>
+    <symbol name="CAN_MODATAH26" address="0xf0004754" entry="false"/>
+    <symbol name="CAN_MODATAH27" address="0xf0004774" entry="false"/>
+    <symbol name="CAN_MODATAH28" address="0xf0004794" entry="false"/>
+    <symbol name="CAN_MODATAH29" address="0xf00047b4" entry="false"/>
+    <symbol name="CAN_MODATAH30" address="0xf00047d4" entry="false"/>
+    <symbol name="CAN_MODATAH31" address="0xf00047f4" entry="false"/>
+    <symbol name="CAN_MODATAH32" address="0xf0004814" entry="false"/>
+    <symbol name="CAN_MODATAH33" address="0xf0004834" entry="false"/>
+    <symbol name="CAN_MODATAH34" address="0xf0004854" entry="false"/>
+    <symbol name="CAN_MODATAH35" address="0xf0004874" entry="false"/>
+    <symbol name="CAN_MODATAH36" address="0xf0004894" entry="false"/>
+    <symbol name="CAN_MODATAH37" address="0xf00048b4" entry="false"/>
+    <symbol name="CAN_MODATAH38" address="0xf00048d4" entry="false"/>
+    <symbol name="CAN_MODATAH39" address="0xf00048f4" entry="false"/>
+    <symbol name="CAN_MODATAH40" address="0xf0004914" entry="false"/>
+    <symbol name="CAN_MODATAH41" address="0xf0004934" entry="false"/>
+    <symbol name="CAN_MODATAH42" address="0xf0004954" entry="false"/>
+    <symbol name="CAN_MODATAH43" address="0xf0004974" entry="false"/>
+    <symbol name="CAN_MODATAH44" address="0xf0004994" entry="false"/>
+    <symbol name="CAN_MODATAH45" address="0xf00049b4" entry="false"/>
+    <symbol name="CAN_MODATAH46" address="0xf00049d4" entry="false"/>
+    <symbol name="CAN_MODATAH47" address="0xf00049f4" entry="false"/>
+    <symbol name="CAN_MODATAH48" address="0xf0004a14" entry="false"/>
+    <symbol name="CAN_MODATAH49" address="0xf0004a34" entry="false"/>
+    <symbol name="CAN_MODATAH50" address="0xf0004a54" entry="false"/>
+    <symbol name="CAN_MODATAH51" address="0xf0004a74" entry="false"/>
+    <symbol name="CAN_MODATAH52" address="0xf0004a94" entry="false"/>
+    <symbol name="CAN_MODATAH53" address="0xf0004ab4" entry="false"/>
+    <symbol name="CAN_MODATAH54" address="0xf0004ad4" entry="false"/>
+    <symbol name="CAN_MODATAH55" address="0xf0004af4" entry="false"/>
+    <symbol name="CAN_MODATAH56" address="0xf0004b14" entry="false"/>
+    <symbol name="CAN_MODATAH57" address="0xf0004b34" entry="false"/>
+    <symbol name="CAN_MODATAH58" address="0xf0004b54" entry="false"/>
+    <symbol name="CAN_MODATAH59" address="0xf0004b74" entry="false"/>
+    <symbol name="CAN_MODATAH60" address="0xf0004b94" entry="false"/>
+    <symbol name="CAN_MODATAH61" address="0xf0004bb4" entry="false"/>
+    <symbol name="CAN_MODATAH62" address="0xf0004bd4" entry="false"/>
+    <symbol name="CAN_MODATAH63" address="0xf0004bf4" entry="false"/>
+    <symbol name="CAN_MOAR0" address="0xf0004418" entry="false"/>
+    <symbol name="CAN_MOAR1" address="0xf0004438" entry="false"/>
+    <symbol name="CAN_MOAR2" address="0xf0004458" entry="false"/>
+    <symbol name="CAN_MOAR3" address="0xf0004478" entry="false"/>
+    <symbol name="CAN_MOAR4" address="0xf0004498" entry="false"/>
+    <symbol name="CAN_MOAR5" address="0xf00044b8" entry="false"/>
+    <symbol name="CAN_MOAR6" address="0xf00044d8" entry="false"/>
+    <symbol name="CAN_MOAR7" address="0xf00044f8" entry="false"/>
+    <symbol name="CAN_MOAR8" address="0xf0004518" entry="false"/>
+    <symbol name="CAN_MOAR9" address="0xf0004538" entry="false"/>
+    <symbol name="CAN_MOAR10" address="0xf0004558" entry="false"/>
+    <symbol name="CAN_MOAR11" address="0xf0004578" entry="false"/>
+    <symbol name="CAN_MOAR12" address="0xf0004598" entry="false"/>
+    <symbol name="CAN_MOAR13" address="0xf00045b8" entry="false"/>
+    <symbol name="CAN_MOAR14" address="0xf00045d8" entry="false"/>
+    <symbol name="CAN_MOAR15" address="0xf00045f8" entry="false"/>
+    <symbol name="CAN_MOAR16" address="0xf0004618" entry="false"/>
+    <symbol name="CAN_MOAR17" address="0xf0004638" entry="false"/>
+    <symbol name="CAN_MOAR18" address="0xf0004658" entry="false"/>
+    <symbol name="CAN_MOAR19" address="0xf0004678" entry="false"/>
+    <symbol name="CAN_MOAR20" address="0xf0004698" entry="false"/>
+    <symbol name="CAN_MOAR21" address="0xf00046b8" entry="false"/>
+    <symbol name="CAN_MOAR22" address="0xf00046d8" entry="false"/>
+    <symbol name="CAN_MOAR23" address="0xf00046f8" entry="false"/>
+    <symbol name="CAN_MOAR24" address="0xf0004718" entry="false"/>
+    <symbol name="CAN_MOAR25" address="0xf0004738" entry="false"/>
+    <symbol name="CAN_MOAR26" address="0xf0004758" entry="false"/>
+    <symbol name="CAN_MOAR27" address="0xf0004778" entry="false"/>
+    <symbol name="CAN_MOAR28" address="0xf0004798" entry="false"/>
+    <symbol name="CAN_MOAR29" address="0xf00047b8" entry="false"/>
+    <symbol name="CAN_MOAR30" address="0xf00047d8" entry="false"/>
+    <symbol name="CAN_MOAR31" address="0xf00047f8" entry="false"/>
+    <symbol name="CAN_MOAR32" address="0xf0004818" entry="false"/>
+    <symbol name="CAN_MOAR33" address="0xf0004838" entry="false"/>
+    <symbol name="CAN_MOAR34" address="0xf0004858" entry="false"/>
+    <symbol name="CAN_MOAR35" address="0xf0004878" entry="false"/>
+    <symbol name="CAN_MOAR36" address="0xf0004898" entry="false"/>
+    <symbol name="CAN_MOAR37" address="0xf00048b8" entry="false"/>
+    <symbol name="CAN_MOAR38" address="0xf00048d8" entry="false"/>
+    <symbol name="CAN_MOAR39" address="0xf00048f8" entry="false"/>
+    <symbol name="CAN_MOAR40" address="0xf0004918" entry="false"/>
+    <symbol name="CAN_MOAR41" address="0xf0004938" entry="false"/>
+    <symbol name="CAN_MOAR42" address="0xf0004958" entry="false"/>
+    <symbol name="CAN_MOAR43" address="0xf0004978" entry="false"/>
+    <symbol name="CAN_MOAR44" address="0xf0004998" entry="false"/>
+    <symbol name="CAN_MOAR45" address="0xf00049b8" entry="false"/>
+    <symbol name="CAN_MOAR46" address="0xf00049d8" entry="false"/>
+    <symbol name="CAN_MOAR47" address="0xf00049f8" entry="false"/>
+    <symbol name="CAN_MOAR48" address="0xf0004a18" entry="false"/>
+    <symbol name="CAN_MOAR49" address="0xf0004a38" entry="false"/>
+    <symbol name="CAN_MOAR50" address="0xf0004a58" entry="false"/>
+    <symbol name="CAN_MOAR51" address="0xf0004a78" entry="false"/>
+    <symbol name="CAN_MOAR52" address="0xf0004a98" entry="false"/>
+    <symbol name="CAN_MOAR53" address="0xf0004ab8" entry="false"/>
+    <symbol name="CAN_MOAR54" address="0xf0004ad8" entry="false"/>
+    <symbol name="CAN_MOAR55" address="0xf0004af8" entry="false"/>
+    <symbol name="CAN_MOAR56" address="0xf0004b18" entry="false"/>
+    <symbol name="CAN_MOAR57" address="0xf0004b38" entry="false"/>
+    <symbol name="CAN_MOAR58" address="0xf0004b58" entry="false"/>
+    <symbol name="CAN_MOAR59" address="0xf0004b78" entry="false"/>
+    <symbol name="CAN_MOAR60" address="0xf0004b98" entry="false"/>
+    <symbol name="CAN_MOAR61" address="0xf0004bb8" entry="false"/>
+    <symbol name="CAN_MOAR62" address="0xf0004bd8" entry="false"/>
+    <symbol name="CAN_MOAR63" address="0xf0004bf8" entry="false"/>
+    <symbol name="CAN_MOCTR0" address="0xf000441c" entry="false"/>
+    <symbol name="CAN_MOCTR1" address="0xf000443c" entry="false"/>
+    <symbol name="CAN_MOCTR2" address="0xf000445c" entry="false"/>
+    <symbol name="CAN_MOCTR3" address="0xf000447c" entry="false"/>
+    <symbol name="CAN_MOCTR4" address="0xf000449c" entry="false"/>
+    <symbol name="CAN_MOCTR5" address="0xf00044bc" entry="false"/>
+    <symbol name="CAN_MOCTR6" address="0xf00044dc" entry="false"/>
+    <symbol name="CAN_MOCTR7" address="0xf00044fc" entry="false"/>
+    <symbol name="CAN_MOCTR8" address="0xf000451c" entry="false"/>
+    <symbol name="CAN_MOCTR9" address="0xf000453c" entry="false"/>
+    <symbol name="CAN_MOCTR10" address="0xf000455c" entry="false"/>
+    <symbol name="CAN_MOCTR11" address="0xf000457c" entry="false"/>
+    <symbol name="CAN_MOCTR12" address="0xf000459c" entry="false"/>
+    <symbol name="CAN_MOCTR13" address="0xf00045bc" entry="false"/>
+    <symbol name="CAN_MOCTR14" address="0xf00045dc" entry="false"/>
+    <symbol name="CAN_MOCTR15" address="0xf00045fc" entry="false"/>
+    <symbol name="CAN_MOCTR16" address="0xf000461c" entry="false"/>
+    <symbol name="CAN_MOCTR17" address="0xf000463c" entry="false"/>
+    <symbol name="CAN_MOCTR18" address="0xf000465c" entry="false"/>
+    <symbol name="CAN_MOCTR19" address="0xf000467c" entry="false"/>
+    <symbol name="CAN_MOCTR20" address="0xf000469c" entry="false"/>
+    <symbol name="CAN_MOCTR21" address="0xf00046bc" entry="false"/>
+    <symbol name="CAN_MOCTR22" address="0xf00046dc" entry="false"/>
+    <symbol name="CAN_MOCTR23" address="0xf00046fc" entry="false"/>
+    <symbol name="CAN_MOCTR24" address="0xf000471c" entry="false"/>
+    <symbol name="CAN_MOCTR25" address="0xf000473c" entry="false"/>
+    <symbol name="CAN_MOCTR26" address="0xf000475c" entry="false"/>
+    <symbol name="CAN_MOCTR27" address="0xf000477c" entry="false"/>
+    <symbol name="CAN_MOCTR28" address="0xf000479c" entry="false"/>
+    <symbol name="CAN_MOCTR29" address="0xf00047bc" entry="false"/>
+    <symbol name="CAN_MOCTR30" address="0xf00047dc" entry="false"/>
+    <symbol name="CAN_MOCTR31" address="0xf00047fc" entry="false"/>
+    <symbol name="CAN_MOCTR32" address="0xf000481c" entry="false"/>
+    <symbol name="CAN_MOCTR33" address="0xf000483c" entry="false"/>
+    <symbol name="CAN_MOCTR34" address="0xf000485c" entry="false"/>
+    <symbol name="CAN_MOCTR35" address="0xf000487c" entry="false"/>
+    <symbol name="CAN_MOCTR36" address="0xf000489c" entry="false"/>
+    <symbol name="CAN_MOCTR37" address="0xf00048bc" entry="false"/>
+    <symbol name="CAN_MOCTR38" address="0xf00048dc" entry="false"/>
+    <symbol name="CAN_MOCTR39" address="0xf00048fc" entry="false"/>
+    <symbol name="CAN_MOCTR40" address="0xf000491c" entry="false"/>
+    <symbol name="CAN_MOCTR41" address="0xf000493c" entry="false"/>
+    <symbol name="CAN_MOCTR42" address="0xf000495c" entry="false"/>
+    <symbol name="CAN_MOCTR43" address="0xf000497c" entry="false"/>
+    <symbol name="CAN_MOCTR44" address="0xf000499c" entry="false"/>
+    <symbol name="CAN_MOCTR45" address="0xf00049bc" entry="false"/>
+    <symbol name="CAN_MOCTR46" address="0xf00049dc" entry="false"/>
+    <symbol name="CAN_MOCTR47" address="0xf00049fc" entry="false"/>
+    <symbol name="CAN_MOCTR48" address="0xf0004a1c" entry="false"/>
+    <symbol name="CAN_MOCTR49" address="0xf0004a3c" entry="false"/>
+    <symbol name="CAN_MOCTR50" address="0xf0004a5c" entry="false"/>
+    <symbol name="CAN_MOCTR51" address="0xf0004a7c" entry="false"/>
+    <symbol name="CAN_MOCTR52" address="0xf0004a9c" entry="false"/>
+    <symbol name="CAN_MOCTR53" address="0xf0004abc" entry="false"/>
+    <symbol name="CAN_MOCTR54" address="0xf0004adc" entry="false"/>
+    <symbol name="CAN_MOCTR55" address="0xf0004afc" entry="false"/>
+    <symbol name="CAN_MOCTR56" address="0xf0004b1c" entry="false"/>
+    <symbol name="CAN_MOCTR57" address="0xf0004b3c" entry="false"/>
+    <symbol name="CAN_MOCTR58" address="0xf0004b5c" entry="false"/>
+    <symbol name="CAN_MOCTR59" address="0xf0004b7c" entry="false"/>
+    <symbol name="CAN_MOCTR60" address="0xf0004b9c" entry="false"/>
+    <symbol name="CAN_MOCTR61" address="0xf0004bbc" entry="false"/>
+    <symbol name="CAN_MOCTR62" address="0xf0004bdc" entry="false"/>
+    <symbol name="CAN_MOCTR63" address="0xf0004bfc" entry="false"/>
     <symbol name="PCP_RAM_BASE" address="0xf0050000" entry="false"/>
     <symbol name="PCP_CLC" address="0xf0043f00" entry="false"/>
     <symbol name="PCP_ID" address="0xf0043f08" entry="false"/>
-- 
2.41.0

